src/components/organism/ShopProducts/OrderPage/CardPay/CardPay.js

```diff
@@ -68,9 +66,9 @@ const CardPay = () => {
         </li>
       </ul>
       <button
-        disabled={buttonDisabled}
+        disabled={isEmptyOrders}
```

Comment:
We may also want to disable button after the user submits the order, until the data is successfully saved in database. To do it, we can use `isSaving` from `useSetDataWithAuth` :)

```
const [setData, , isSaving] = useSetDataWithAuth();
...
<button
     disabled={isEmptyOrders}
     type="button"
     className={(isEmptyOrders || isSaving) ? 'btn-disabled' : 'btn-order'}